site/docs/pages/wallet/wallet.mdx

```diff
@@ -186,6 +190,44 @@ You can override component text using `text`.
   </Wallet>
 </WalletComponents>
 
+### Add Basenames support
```

Comment:
Let's have this in his own page.

Reply:
TODO: I will fast follow with adding a separate page under `Wallet` for `WalletDropdownBaseName`, `walletDropdownLink` and `WalletDropdownDisconnect`